test/concatenate5.cc

```diff
@@ -14,6 +14,7 @@
 #include <random>
 #include <vector>
 
+#include <fp16.h>
```

Comment:
Please all test fp16 includes to #include <fp16/fp16.h>

Reply:
This may be the only one you have to fix, it's the only one failing to build internally